src/components/common/string/StringField.tsx

```diff
@@ -12,41 +12,42 @@ export interface StringFieldProps {
   label: string;
   fieldValue: string;
   canEdit: boolean;
+  onStringValueChange: Function;
   required?: boolean;
 }
 
 const StringField = ({
```

Comment:
Can we change the name of the component to StringInput just to be consistent with all other primitive components?

Reply:
thanks for the comment, I'll change it